target/linux/ramips/dts/BARIONET-1000-V1-1-A9.dts

```diff
@@ -0,0 +1,145 @@
+/dts-v1/;
+
+#include "mt7620a.dtsi"
+
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "BARIONET-1000-V1-1-A9", "ralink,mt7620a-soc";
```

Comment:
Is it really necessary to include the full version number for the board (question applies as well as to model device tree property and boardname in userspace)?

Nevertheless, should be `compatible = "barix,barionet1000-v1", "ralink,mt7620a-soc";` or similar